src/windows/custom_currency.c

```diff
@@ -0,0 +1,287 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+/*****************************************************************************
+ * OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+ *
+ * OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+ * For more information, visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include "../localisation/localisation.h"
+#include "../interface/widget.h"
+#include "../interface/window.h"
+#include "../config.h"
+#include "dropdown.h"
+
+enum WINDOW_CUSTOM_CURRENCY_WIDGET_IDX {
+    WIDX_BACKGROUND,
```

Comment:
The [Coding style](https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-style) page of teh wiki has been edited recently (June, 2), maybe that's causing the confusion.

It says that following these coding standards is not a priority. I considered replacing tabs by spaces important because spaces have a fixed width (tabs, depending on the editor configuration, can leave very much blank space in the code).

I don't think these changes are difficult to read but, anyway, if this is considered an important thing, I can change that, no problem.